Tests/MongoSwiftSyncTests/RetryableReadsTests.swift

```diff
@@ -42,11 +42,11 @@ private struct RetryableReadsTestFile: Decodable, SpecTestFile {
     let tests: [RetryableReadsTest]
 
     static let skippedTestFileNameKeywords = [
-        "changeStream", // TODO: SWIFT-648: Unskip this test
```

Comment:
I went ahead and moved this ticket into code review so we remember to close it once this PR is merged.

Reply:
thanks! I forgot to go do that.